packages/@react-types/layout/src/flex.d.ts

```diff
@@ -13,6 +13,9 @@
 import {DOMProps, FlexStyleProps} from '@react-types/shared';
 import {ReactElement} from 'react';
 
+export type Slots = {[key: string]: any};
```

Comment:
Since grid and flex both have this definition should I move this somewhere common?